tools/Docs/az-ps-latest.csv

```diff
@@ -135,7 +135,7 @@ pac133,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-rele
 pac134,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-release/Az.SqlVirtualMachine.1.1.0.nupkg;sourceType=sa]Az.SqlVirtualMachine,1.1.0
 pac135,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-release/Az.Ssh.0.1.1.nupkg;sourceType=sa]Az.Ssh,0.1.1
 pac136,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-release/Az.StackHCI.1.4.3.nupkg;sourceType=sa]Az.StackHCI,1.4.3
-pac137,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-release/Az.Storage.5.4.1.nupkg;sourceType=sa]Az.Storage,5.4.1
+pac137,[ps=true;customSource=https://azpspackage.blob.core.windows.net/docs-release/Az.Storage.5.4.2.nupkg;sourceType=sa]Az.Storage,5.4.2
```

Comment:
Question: What's this file for?
It looks we don't need to change this file for preview release before.